mp/src/game/client/momentum/ui/gameui/loadingscreen/LoadingScreen.cpp

```diff
@@ -224,9 +225,12 @@ void CLoadingScreen::OnKeyCodeTyped(KeyCode code)
 
 void CLoadingScreen::SetTipPanelText(const char *pMapName)
 {
-    const auto pFound = g_pGameModeSystem->GetGameModeFromMapName(pMapName);
-    const auto eMode = pFound ? pFound->GetType() : GAMEMODE_UNKNOWN;
-    m_pTipLabel->SetText(m_TipManager.GetTipForGamemode(eMode));
+    const auto eMode = mom_gamemode_override.GetBool() ?
+        g_pGameModeSystem->GetGameMode(mom_gamemode_override.GetInt())->GetType()
+        :
+        g_pGameModeSystem->GetGameModeFromMapName(pMapName)->GetType();
+    
+    m_pTipLabel->SetText(m_TipManager.GetTipForGamemode(eMode ? eMode : GAMEMODE_UNKNOWN));
```

Comment:
eMode is confirmed to be a valid gamemode here. The old ternary logic was a null check, which is no longer needed.
```suggestion
    m_pTipLabel->SetText(m_TipManager.GetTipForGamemode(eMode);
```

Reply:
And on second checking, it seems like you reverted the change to `GetGameModeFromMapName` to return the null gamemode instead of the unknown one. I liked that change and it should be used instead, otherwise this is most likely going to null pointer on an unknown map now.